### DISCUSSION TITLE
Typo in Getting Started wiki page

### DISCUSSION THREAD

**Discussion by HongGers** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3501] [General]
Hi all.
I found an issue in [Getting Started -> Version 5.0.0 and later section](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Getting-Started#version-500-and-later). It seems like there is a redundant slash("/") in the first example import code.

original:
`<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes//MaterialDesign3.Defaults.xaml"/> `
should be:
`<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign3.Defaults.xaml"/> `

I can use the library only if I delete the redundant slash. or I'll get a "cannot find resources" error. Is that a typo?

_I'm not sure which kind of issue to open. so I decided to open a discussion._
_I'm new to github and contribution. if I'm doing something incorrectly. Just tell me. Thanks!_

**Comment by Keboo** [comment]
Thank you for reporting this. That was just a typo from recent updates for 5.0.0 work. Fixed now.
